lib/stream/tag.rb

```diff
@@ -29,7 +29,12 @@ def tagged_people_count
   end
 
   def posts
-    @posts ||= construct_post_query
+    tag_id = [tag.try(:id)]
```

Comment:
Yes, that's the hack. Before, it worked with a non-existent tag name and did simply not return results. Now I need a tag ID and retrieving that may fail.